client/blocks/reader-share/style.scss

```diff
@@ -1,18 +1,29 @@
-.reader-share__button {
-	align-items: center;
-	box-sizing: border-box;
+.reader-share__button.button {
 	color: var( --color-text-subtle );
-	display: inline-flex;
 	padding: 4px;
 	position: relative;
+	display: inline-flex;
+	align-items: center;
 
 	&:hover,
-	&:focus,
 	&:active {
-		cursor: pointer;
 		color: var( --color-primary );
 	}
 
+	&:focus {
+		outline: thin dotted;
+	}
+
+	&.is-borderless {
+		.gridicon {
+			width: unset;
```

Comment:
needs testing in IE, unsupported there

Reply:
Great catch!
I changed the implementation a little bit to correctly support this case in d411427, considering that `Button` overrides the `width` and `height` properties of child `Gridicon` components and unsetting the properties is not an option.

The trade-off in this case is that the `iconSize` property is mapped to the `Button`'s `compact` property. The `compact` property sets the size of the icon to `18px`, so when `iconSize` is `18`, `compact` is set to `true`. Otherwise, the size is `24px`. I verified in the two components that use the `ReaderShare`component  (indirectly through `ReaderPostActions` component), and these sizes match the ones that were passed to it.

Tested it with Internet Explorer and it's working now 👍